### ISSUE TITLE
代码中问题

### ISSUE THREAD

**Issue by purse1996** [earhian/Humpback-Whale-Identification-1st-#12]
您好 我们想在您的基础上训练一个分类网络，但细节的地方有两个位置不太理解，能不能麻烦解释一下，万分感谢。
第一个是results为什么后面还要torch.cat一个torch.ones_like
results = torch.cat([torch.sigmoid(results), torch.ones_like(results[:, :1]).float().cuda() * 0.5], 1)
第二个是这里为什么要乘以16
out = self.fc(global_feat) * 16

**Comment by earhian** [comment]
因为我对特征做了一个归一化，值都会比较小，同时也不想让参数的范围过大，所以就乘了一个系数
